mkdocs/docs/HPC/setting_up_python_virtual_environments.md

```diff
@@ -0,0 +1,262 @@
+# Python Virtual Environments (venv's)
+
+## Introduction
+
+A Python virtual environment (venv for short)
+is a tool to create an isolated Python workspace with its own dependencies.
```

Comment:
I meant packages, but I see how it would be confusing. Removed in [removed 'dependencies' from sentence](https://github.com/hpcugent/vsc_user_docs/pull/639/commits/2bc34af88ea4cd0bbcecaf9b5f6faa373cd0396b)